docs/guides/deleting-series.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Deleting Series"
+linkTitle: "Deleting Series"
+weight: 3
+slug: deleting-series
+---
+
+:warning: This feature is currently experimental and is only supported for Chunks storage. :warning:
+
+Cortex supports deletion of series using [Prometheus compatible API](https://prometheus.io/docs/prometheus/latest/querying/api/#delete-series).
+It however does not support [Prometheuses Clean Tombstones](https://prometheus.io/docs/prometheus/latest/querying/api/#clean-tombstones) API by trading it in for another feature for allowing cancellation of delete requests.
+
+### How it works
+
+A new service called `purger` is added which handles deletion APIs and does the processing of the requests.
+To store the requests, and some additional information while performing deletion it requires configuring an index and object store respectively for it.
+`purger` can be configured using the config defined [here](../configuration/config-file-reference.md#purger_config)
+
+All the requests specified below needs to be sent to `purger`.
+
+**Note:** If you have enabled auth then same as other APIs it is required to have `X-Scope-OrgID` header set in the requests.
+
+#### Requesting Deletion
+
+By calling the `delete_series` API defined [here](https://prometheus.io/docs/prometheus/latest/querying/api/#delete-series), you can request the deletion of series.
```

Comment:
A couple of suggestions:
1. Instead of `delete_series` mention the exact endpoint for clarity (eg. `/api/v1/admin/tsdb/delete_series`)
2. Try to avoid to link "here" and instead link some words related to the actual content in the link (eg. "here" > "Prometheus deletion API")